src/test/threads/test_threads_noexit.rs

```diff
@@ -0,0 +1,21 @@
+/*
+ * The Shadow Simulator
+ * See LICENSE for licensing information
+ */
+
+use std::error::Error;
+
+const NUM_SECONDS: u64 = 3;
```

Comment:
Worth a comment that this must be > stoptime in the test config for this test to be meaningful